stdlib/public/core/ArrayShared.swift

```diff
@@ -12,9 +12,10 @@
 /// This type is used as a result of the `_checkSubscript` call to associate the
 /// call with the array access call it guards.
 ///
-/// In order for the optimizer see that a call to `_checkSubscript` is semantically
-/// associated with an array access, a value of this type is returned and later passed
-/// to the accessing function.  For example, a typical call to `_getElement` looks like
+/// In order for the optimizer see that a call to `_checkSubscript`
+/// is semantically associated with an array access, a value of this type
+/// is returned and later passed to the accessing function.
+/// For example, a typical call to `_getElement` looks like
 ///   let token = _checkSubscript(index, ...)
 ///   return _getElement(index, ... , matchingSubscriptCheck: token)
```

Comment:
Did you mean to land this? If so, it might be a good idea to fix the markup, too:
```suggestion
///
///     let token = _checkSubscript(index, ...)
///     return _getElement(index, ... , matchingSubscriptCheck: token)
```

AIUI the code sample currently gets typeset as if it was part of the preceding paragraph.

Reply:
Wow, that was a myopic change from me. Thanks for looking at the very next line!